server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -218,15 +221,50 @@ private TrTag renderTIRow(Account ti) {
         .with(renderStatusCell(ti));
   }
 
-  private TrTag renderApplicantRow(Account applicant) {
+  private TrTag renderApplicantRow(
+      Account applicant, TrustedIntermediaryGroup tiGroup, Http.Request request) {
     return tr().withClasses(
             ReferenceClasses.ADMIN_QUESTION_TABLE_ROW,
             Styles.BORDER_B,
             Styles.BORDER_GRAY_300,
             StyleUtils.even(Styles.BG_GRAY_100))
         .with(renderInfoCell(applicant))
         .with(renderApplicantInfoCell(applicant))
-        .with(renderActionsCell(applicant));
+        .with(renderActionsCell(applicant))
+        .with(renderDateOfBirthCell(applicant, tiGroup, request));
+  }
+
+  private TdTag renderDateOfBirthCell(
+      Account account, TrustedIntermediaryGroup tiGroup, Http.Request request) {
+
+    Optional<Applicant> newestApplicant = account.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12);
+    }
+    Optional<LocalDate> current_DOB = newestApplicant.get().getApplicantData().getDateOfBirth();
+    // for cases where DOB is already present, display the Date of Birth
+    // for other cases, show the form to update the DOB.
+    // Note: the update can be done only once.
+    if (current_DOB.isPresent()) {
+      LocalDate unformatedDate = current_DOB.get();
+      return td().with(
+              div(unformatedDate.format(DateTimeFormatter.ofPattern("MM-dd-yyyy")))
```

Comment:
suggestion (blocking): use [DateConverter](https://github.com/seattle-uat/civiform/blob/main/server/app/services/DateConverter.java#L14)

Reply:
Not needed at it is binding with the input() tag